### PR TITLE
Add ratio comparison

### PR DESCRIPTION
**Pull request by mayerm94**
Add ratio comparison to comparison.coffee so it calls ratio's already-existing equals function

Pull requests into cql-execution require the following.
Submitter and reviewer should ✔ when done.
For items that are not-applicable, mark "N/A" and ✔.

[CDS Connect](https://cds.ahrq.gov/cdsconnect) and [Bonnie](https://github.com/projecttacoma/bonnie) are the main users of this repository. 
It is strongly recommended to include a person from each of those projects as a reviewer.

**Submitter:**
- [x] This pull request describes why these changes were made
- [x] Code diff has been done and been reviewed (it does not contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [] Code coverage has not gone down and all code touched or added is covered.
- [x] All dependent libraries are appropriately updated or have a corresponding PR related to this change

**Reviewer:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: src/elm/ratio.coffee [hunk 1]
```diff
@@ -36,5 +36,9 @@ module.exports.Ratio = class Ratio extends Expression
       divided_other = other.numerator.dividedBy(other.denominator)
       divided_this.equals(divided_other)
 
+  equivalent: (other) ->
+    if other instanceof Ratio
+      @equals(other)
```
Comment (jbradl11) [inline src/elm/ratio.coffee]: This returns null if `other` is not a Ratio correct, was that what was decided?

Comment (jbradl11) [reply src/elm/ratio.coffee]: Also, if this just calls the equals class in Ratio, that also does a check for `other instanceof Ratio` so I don't think you need a duplicate check here, unless we want to return a different "falsey" value for equivalence.

**Comment by cmoesel** [comment]
OK.  Good news (for us).  CQL-to-ELM doesn't allow `<`, `<=`, `>`, `>=` with `Ratio`.  If you try it, you get an error like this:
```
<data.coffee#Less[16:20, 16:53]> Could not resolve call to operator Less with signature (System.Ratio,System.Ratio).
```

CQL-to-ELM _does_ support `=` and `~` though.  You may want to consider adding tests for Ratio in `test/elm/comparison/`.  For example:
```
define EqRatios: 10 'mg' : 2 'dL' = 15 'mg' : 3 'dL'
define UneqRatios: 10 'mg' : 2 'dL' = 15 'mg' : 4 'dL'
```
and
```
define EqRatios: 10 'mg' : 2 'dL' ~ 15 'mg' : 3 'dL'
define UneqRatios: 10 'mg' : 2 'dL' ~ 15 'mg' : 4 'dL'
```

File: test/elm/comparison/test.coffee [hunk 2]
```diff
@@ -93,6 +93,12 @@ describe 'Equal', ->
   it 'should be null for 5 Cel = 40 m', ->
     should(@aLtB_Quantity_incompatible.exec(@ctx)).be.null()
 
+  it 'should be true for 10mg:2dL = 15mg:3dL', ->
+    @eqRatios.exec(@ctx).should.be.true()
+  
+  it 'should be true for 10mg:2dL = 15mg:4dL', ->
```
Comment (cmoesel) [inline test/elm/comparison/test.coffee]: typo: it should be _false_

File: test/elm/comparison/test.coffee [hunk 3]
```diff
@@ -197,7 +203,13 @@ describe 'Equivalent', ->
 
   it 'should be true for FOO ~ foo', ->
     @caseInsensitiveStrings.exec(@ctx).should.be.true()
+
+  it 'should be true for 10mg:2dL ~ 15mg:3dL', ->
+    @eqRatios.exec(@ctx).should.be.true()
   
+  it 'should be true for 10mg:2dL ~ 15mg:4dL', ->
```
Comment (cmoesel) [inline test/elm/comparison/test.coffee]: typo: it should be _false_

File: test/elm/comparison/test.coffee (hunk 2 above)
Comment (mayerm94) [reply test/elm/comparison/test.coffee]: <facepalm> I shouldn't have let that go through
